date_range/models/date_range.py

```diff
@@ -8,7 +8,7 @@
 class DateRange(models.Model):
     _name = "date.range"
     _description = "Date Range"
-    _order = "type_name, date_start"
+    _order = "type_id, date_start"
```

Comment:
`type_id` destination model has name order?

Reply:
No, the date.range.type model does not have any sorting parameters established, but I am not sure that it can be sorted by name either, since it is a translated field, I am not sure how Postgres will interpret that when it is stored as a JSON column.
I also don't see the fact that they are ordered by name as something very important.

Reply:
Another option is that this field is not translatable, so it can be left as it was.

Reply:
I think it's worth to try to put such order and see if it works.

Reply:
I really didn't expect this, because it really orders it correctly and to each user in their language.